consuming/visualization/terminal_viz.py

```diff
@@ -0,0 +1,367 @@
+import logbook.configuration as log
+
+class MainScreen():
+    # Where to store them best? 7   '0123456'
+    mapIdentifiers = {'Empty':      '       ',
```

Comment:
So, MainScreen should have just a few properties and methoods like Render(), CleanScr(), and so on... Also, if you want to follow MVCVM pattern, or stick close to it, it MUST have its own Model and ViewModel. Then, such a mapping would move to MainScreenModel, or even better, instance of the class mentioned in previous comment should be referencied from MainScreenViewModel.
Nevertheless, Im not sure that MainScreen class has anything in common with Details representation. From my point of view, MainScreen should operate on Windows and take care only about them. Thus, the Window MVCVM would take care about rendering Detail depending on for.ex. Zoom, window status and so on. In this case, the base class WindowViewModel should have a reference to INDEPENDENT instance of  some class which has such a mapping as properties.